docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -64,7 +64,7 @@ The Trafodion bulk load process takes place in two phases:
 * _Preparation phase_: In this phase, Trafodion reads the data from
 the source files in Hive or HDFS, partitions the data based on the
 target table's partitioning scheme, sorts the data, and then generates
-KeyValue pairs that will populate the HFiles. Trafodion also encodes the
+Key-value pairs that will populate the HFiles. Trafodion also encodes the
```

Comment:
I suspect the old text is referring to a Java class "KeyValue" (but then "KeyValue pairs" would not be correct, "KeyValue objects" would be better). I think it is OK to change this to "key-value" (referring to a concept rather than a Java class), but in that case it shouldn't be capitalized.